src/SupportedVideoProviders.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\itk_video;
+
+/**
+ * Supported video providers.
+ */
+enum SupportedVideoProviders: string {
+  case VIDEO_TOOL = 'video_tool';
+  case VIMEO = 'vimeo';
+
+  /**
+   * Get config for supported video providers.
+   *
+   * @return array[]
+   *   A list of configurations for supported video providers.
+   */
+  public static function getConfig(): array {
+    return [
+      'video_tool' => [
```

Comment:
Use the enum values here, e.g.:

```suggestion
      self::VIDEO_TOOL->value => [
```